lua/cfc_entity_stubber/m9k/m9k_assault_rifles/m9k_ak47.lua

```diff
@@ -0,0 +1,8 @@
+AddCSLuaFile( )
```

Comment:
get rid of all these spaces inside parentheses like this

Reply:
idk how to ngl